tyk-stack/templates/NOTES.txt

```diff
@@ -1,7 +1,49 @@
 
-=====================================
-  Manage Ingress resources and APIs
-=====================================
+NOTE:
+{{- $tykApiPort :=  include "tyk-stack.gwServicePort" . -}}
+{{- if index .Values "tyk-dashboard" "dashboard" "tykApiHost" }}
+    Tyk Dashboard connects to Tyk Gateway at {{ index .Values "tyk-dashboard" "dashboard" "tykApiHost"  }} over port {{ $tykApiPort }} for key management functions.
+{{- else }}
+    Tyk Dashboard connects to Tyk Gateway at {{ (include "tyk-dashboard.gw_proto" (index .Subcharts "tyk-dashboard")) }}://{{ (include "tyk-dashboard.gateway_host" (index .Subcharts "tyk-dashboard")) }} over port {{ $tykApiPort }} for key management functions.
+{{- end }}
+    Please double check if Tyk Gateway runs on this address. If not, Tyk Gateway host and Tyk Gateway port can be overridden
+        through .tyk-dashboard.dashboard.tykApiHost and .global.servicePorts.gateway fields respectively in values.yaml.
+
+{{ if index .Values "tyk-gateway" "gateway" "control" "enabled" }}
+{{- $gwSvcName := printf "gateway-control-svc-%v" (include "tyk-gateway.fullname" (index .Subcharts "tyk-gateway"))  -}}
+To quickly test everything is ok, you can port-forward Tyk Gateway pod:
+    kubectl port-forward --namespace {{ .Release.Namespace }} service/{{ $gwSvcName }} {{ index .Values "tyk-gateway" "gateway" "control" "port" }}:{{ index .Values "tyk-gateway" "gateway" "control" "containerPort" }}
+    curl localhost:{{ index .Values "tyk-gateway" "gateway" "control" "port" }}/hello
+{{- else }}
+{{- $gwSvcName := printf "gateway-svc-%v" (include "tyk-gateway.fullname" (index .Subcharts "tyk-gateway"))  -}}
+To quickly test everything is ok, you can port-forward Tyk Gateway pod:
+    kubectl port-forward --namespace {{ .Release.Namespace }} service/{{ $gwSvcName }} {{ index .Values "global" "servicePorts" "gateway" }}:{{ index .Values "global" "servicePorts" "gateway" }}
+    curl localhost:{{ index .Values "global" "servicePorts" "gateway" }}/hello
+{{- end }}
+
+{{ if .Values.global.components.bootstrap -}}
+=== Tyk Dashboard Login Details ===
+{{- if not .Values.global.adminUser.useSecretName }}
+    export TYK_DASHBOARD_ADMINEMAIL={{ .Values.global.adminUser.email }}
+    export TYK_DASHBOARD_ADMINPASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} tyk-dashboard-login-details -o jsonpath="{.data.adminUserPassword}" | base64 --decode)
+{{- else }}
+    export TYK_DASHBOARD_ADMINEMAIL=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ .Values.global.adminUser.useSecretName }} -o jsonpath="{.data.adminUserEmail}" | base64 --decode)
+    export TYK_DASHBOARD_ADMINPASSWORD=$(kubectl get secret --namespace {{ .Release.Namespace }} {{ .Values.global.adminUser.useSecretName }} -o jsonpath="{.data.adminUserPassword}" | base64 --decode)
+{{- end }}
+{{- end }}
+
+{{- if index .Values "tyk-dashboard" "dashboard" "ingress" "enabled" }}
+=== Ingress ===
+{{- range $host := .Values.dashboard.ingress.hosts }}
```

Comment:
Error: template: tyk-stack/templates/NOTES.txt:38:26: executing "tyk-stack/templates/NOTES.txt" at <.Values.dashboard.ingress.hosts>: nil pointer evaluating interface {}.ingress

I got this error while installing

Reply:
fixed in https://github.com/TykTechnologies/tyk-charts/pull/186/commits/5ac80e7d372523277935e01db24dfdd5333e86bc